File: 1312 - R2/problem_1312.cs
using System;

namespace problem_1312
{
    class Program
    {
        static void Main(string[] args)
        {
            String[] separador = new String[2];
            int R1, res, R2;
            String num = Console.ReadLine();
            separador = num.Split(' ');
            R1 = Convert.ToInt32(separador[0]);
            res = Convert.ToInt32(separador[1]);
            R2 = (res*2)-R1;
            Console.WriteLine(R2);
        }
    }
}

File: 1312 - R2/problem_1312.java
import java.util.Scanner;


public class problem_1312 {
    public static void main(String[] args) {
        Scanner e = new Scanner(System.in);
        String[] separador = new String[2];
        int R1, res, R2;
        String num = e.nextLine();
        separador = num.split(" ");
        R1 = Integer.parseInt(separador[0]);
        res = Integer.parseInt(separador[1]);
        R2 = (res*2)-R1;
        System.out.println(R2);
    }
}
